### PR TITLE
Adding Kristen's profile

### PR DESCRIPTION
**Pull request by KristenHu**

### CODE REVIEW

File: Kristen-Hu/Kristen-Hu.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+id: "05daa2a9-8768-446e-b9d3-580c8a6f9b8b"
+
+category: "Developers"
+role: "Sample Profile"
```
Comment (SebastienBoissiere) [inline Kristen-Hu/Kristen-Hu.md:5]: Can you replace "Sample Profile" with your job title

**Review by SebastienBoissiere — CHANGES_REQUESTED** [review]
Please delete the cat_computer.jpg file

File: Kristen-Hu/Kristen-Hu.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+id: "05daa2a9-8768-446e-b9d3-580c8a6f9b8b"
```
Comment (SebastienBoissiere) [inline Kristen-Hu/Kristen-Hu.md:2]: Your userid is incorrect. It should be 31aa42cd-719c-e911-825f-00155d01ef64

Comment (KristenHu) [reply Kristen-Hu/Kristen-Hu.md]: Done.

File: Kristen-Hu/Kristen-Hu.md (hunk 1 above)
Comment (KristenHu) [reply Kristen-Hu/Kristen-Hu.md]: Done.

**Review by SebastienBoissiere — APPROVED** [review]

**Pull request merged**